src/rx5808-pro-diversity/settings.h

```diff
@@ -83,6 +89,13 @@ SOFTWARE.
     #define DIVERSITY_MAX_CHECKS 5
 #endif
 
+#ifdef FAST_DIVERSITY_SWITCHING
+    // see https://www.arduino.cc/en/Reference/PortManipulation
+    // PORTC means analog ports (PORTD is arduino 0-7, PORTB is 8-13)
+    // PC0 is the current value of A0 (receiver A) PC1 is value of A1 (receiver B)
+    #define TOGGLE_RECEIVER PORTC ^= (1 << PC0)|(1 << PC1);
```

Comment:
@jelle737 the compiler already does the precalc so it isnt a size/performance problem. i agree that the binary version is probably more readable for most people. 
if we would assume that the switches are all on PORTC we could write is as 
`PORTC ^= (1 << (receiverA_led % 14) ) | (1 << (receiverB_led % 14));`
(A0 is 14 and A1 is 15 on 'regular' arduino)
and it would work with user defined receiverA_led/receiverB_led if they would be on same port (and would be precompiled to B00000011 in the default case)